src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -15,55 +15,47 @@
 package com.google.sps.data;
 
 import com.google.sps.data.Photo;
+import com.google.gson.annotations.SerializedName;
 
 import java.io.Serializable;
 
 public final class Restaurant implements java.io.Serializable {
     private transient int weight = 0;
     private final String name;
     private final double rating;
-    private final String business_status;
+
+    @SerializedName(value = "businessStatus", alternate = "business_status")
```

Comment:
"business_status" -> businessStatus?

Reply:
this is so I don't have to make a custom serializer class. The JSON response calls it "business_status", so this renames it to businessStatus